pkgs/development/python-modules/dipy/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, packaging
+, pytest
+, cython
+, numpy
+, scipy
+, h5py
+, nibabel
+}:
+
+buildPythonPackage rec {
+  pname = "dipy";
+  version = "1.1.1";
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner  = pname;
```

```suggestion
    owner  = "dipy";
```
See https://github.com/NixOS/nixpkgs/pull/78872#discussion_r375371767 for me making the same sort-of-mistake.

---

I don't exactly understand this criticism (you might always in future decide to weaken the set of equalities satisfied by your program; also, the maintainer has intentionally chosen this owner name) but I'm happy to change it.

---

Reverted.

---

ack. Not 100% sure of why not, but it's a small enough style thing that I don't care to fight it, personally.